workspaces/client/app/routes/dashboard/oauth-apps/config/internal/hooks/use-update-managers.ts

```diff
@@ -0,0 +1,34 @@
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { useRepository } from "~/hooks/use-repository";
+import { useToast } from "~/hooks/use-toast";
+
+interface IMutationParams {
+	managerUserIds: string[];
+}
+
+export function useUpdateManagers({ id }: { id: string }) {
+	const { oauthAppsRepository } = useRepository();
+	const { pushToast } = useToast();
+	const queryClient = useQueryClient();
+
+	return useMutation({
+		mutationFn: (payload: IMutationParams) =>
+			oauthAppsRepository.updateManagers(id, payload.managerUserIds),
+		onSuccess: () => {
+			pushToast({
+				type: "success",
+				title: "マネージャーの更新に成功しました",
```

Comment:
```suggestion
				title: "マネージャーを更新しました",
```
ほかのところが成功しましたじゃないからこうかな